app/src/main/res/values/strings.xml

```diff
@@ -607,6 +610,10 @@
     <string name="settings_autostart_bootup_heading">Auto-start on power-up (experimental)</string>
     <string name="settings_autostart_bootup_desc">On device power-up, start RethinkDNS if it was running before shut down.</string>
 
+    <string name="settings_network_header">NETWORK(experimental)</string>
```

Comment:
Network

Add "(experimental)" to `settings_network_all_networks`

Reply:
Modified. Added the experimental tag to `settings_network_all_networks`